pkg/reconciler/builder/builder.go

```diff
@@ -23,6 +12,14 @@ import (
 	"github.com/pivotal/kpack/pkg/reconciler"
 	"github.com/pivotal/kpack/pkg/registry"
 	"github.com/pivotal/kpack/pkg/tracker"
```

Comment:
can we put the kpack packages at the bottom of the import clause similar to before?

Reply:
Done